desc/equilibrium/equilibrium.py

```diff
@@ -2140,6 +2140,7 @@ def solve(
         ----------
         objective : {"force", "forces", "energy"}
             Objective function to solve. Default = force balance on unified grid.
+            ObjectiveFunction or LinearConstraintProjection can also be passed.
```

Comment:
I don't think `LinearConstraintProjection` is public so shouldn't mention it here in a public docstring. Using it here is only for internal purposes anyways.